src/map/entities/charentity.cpp

```diff
@@ -3098,6 +3097,30 @@ bool CCharEntity::OnAttackError(CAttackState& state)
     return false;
 }
 
+bool CCharEntity::isInTriggerArea(uint32 triggerAreaID)
+{
+    if (auto found = charTriggerAreaIDs.find(triggerAreaID); found != charTriggerAreaIDs.end())
+    {
+        return true;
+    }
+    return false;
```

Comment:
`return charTriggerAreaIDs.find(triggerAreaID) != charTriggerAreaIDs.end();`

Reply:
oh wow that is painfully obvious, I'll patch that up later tonight.